internal/image/image_test.go

```diff
@@ -11,6 +11,7 @@ import (
 )
 
 func TestScanImage(t *testing.T) {
+	testutility.SkipIfNotAcceptanceTesting(t, "Not consistent on MacOS/Windows")
```

Comment:
@robramsaynz this is the way to mark tests as run only during acceptance testing. 

It uses [this function](https://github.com/google/osv-scanner/blob/3c2e2ce9cafd6d2e330daa408fee479f6fda2aa0/internal/testutility/utility.go#L45 ) that sets a test property to skip. 

There is also [this function](https://github.com/google/osv-scanner/blob/3c2e2ce9cafd6d2e330daa408fee479f6fda2aa0/internal/testutility/utility.go#L39) for if we just need s switch for some code block rather than to skip the test completely. Which can bee seen used [here](https://github.com/google/osv-scanner/blob/3c2e2ce9cafd6d2e330daa408fee479f6fda2aa0/cmd/osv-scanner/main_test.go#L524) where we create individual local database directories for each test in one case and not in the other.

All of the flow is dependent on the envvar TEST_ACCEPTANCE=true and using the second function as the access point to the envvar.

Reply:
Thanks for the info @billielynch 🙏